tests/integration/test_crnn_integration.py

```
@@ -0,0 +1,192 @@
+        headers={"Content-Type": "application/json"}
+    )
+    request_time = time.time() - start_time
+    
+    if response.status_code == 200:
+        result = response.json()
+        recognized_text = result['text']
+        confidence = result['average_confidence']
+        processing_time = result['processing_time']
+        
+        # 精度計算
+        correct_chars = sum(1 for e, r in zip(expected_text, recognized_text) if e == r)
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)